parity/cli.rs

```diff
@@ -219,6 +219,8 @@ Footprint Options:
                            fast - maintain journal overlay. Fast but 50MB used.
                            auto - use the method most recently synced or
                            default to fast if none synced [default: auto].
+  --pruning-history NUM    Set a number of recent states to keep when pruning
+                           is active. [default: 64].
```

Comment:
They are more stable than ETH currently. ETH has higher uncle rate because of the attack.